python-frontend/src/main/java/org/sonar/python/types/InferredTypes.java

```diff
@@ -153,6 +154,23 @@ public static InferredType runtimeType(@Nullable Symbol typeClass) {
     return anyType();
   }
 
+  public static InferredType anyOrUnknownClassType(Symbol symbol) {
```

Comment:
Probably here as well we need a comment as well explaining that we assume unknown imported symbols whose name start with a capital letter to be classes.